staging/src/k8s.io/apiserver/pkg/endpoints/filters/audit.go

```diff
@@ -56,7 +56,12 @@ func WithAudit(handler http.Handler, sink audit.Sink, policy policy.Checker, lon
 		}
 
 		ev.Stage = auditinternal.StageRequestReceived
-		processAuditEvent(sink, ev, omitStages)
+		if !processAuditEvent(sink, ev, omitStages) {
```

Comment:
`if processed := processAuditEvent(...); !processed` would be clearer.

Reply:
Thanks, done.